library/src/platform/unknown.rs

```diff
@@ -0,0 +1,20 @@
+use crate::UpdateError;
+use anyhow::bail;
+use std::{
+    io::Cursor,
+    path::{Path, PathBuf},
+};
+
+const UNKNOWN_PLATFORM_ERR_MSG: &str = "Unknown platform";
+
+pub fn open_base_lib(_app_dir: &Path, _lib_name: &str) -> anyhow::Result<Cursor<Vec<u8>>> {
+    bail!(UNKNOWN_PLATFORM_ERR_MSG)
```

Comment:
Do the tests still run on a linux host, etc with this?  They must...

Reply:
Yep, they do. The same tests still run (i.e., the Android tests). iOS tests don't currently execute, which, in addition to coverage, is why this is a draft.

This particular code never runs, it's solely so that the package will build in a non-test, non-android, non-ios environment.